### PR TITLE
EXMID <-> ~P 1o e. Fin

### LINKED ISSUE #4110: EXMID <-> ~P 1o e. Fin
It seems very likely that `EXMID <-> ~P 1o e. Fin` can be proved in light of [EXMID ↔ 𝒫 1o = 2o](https://us.metamath.org/ileuni/exmidpweq.html) , [𝒫 1o ≠ 3o](https://us.metamath.org/ileuni/pw1ne3.html), [𝒫 1o ≠ 1o](https://us.metamath.org/ileuni/pw1ne1.html), etc.

### PR DESCRIPTION
**Pull request by jkingdon**
This is a formalization of the proof from https://github.com/metamath/set.mm/issues/4110#issuecomment-2274064019 (I hope I got the mapping from github usernames to contributors correct).

### CODE REVIEW

File: iset.mm [hunk 1]
```diff
@@ -66238,6 +66238,15 @@ the first case of his notation (simple exponentiation) and subscript it
     HVHVEKBVHVQVEUIVSUJUKULSUMVFVOBVFVNAVFVJVKVFVJUNZVMVKWBVIVEVHWBVIVQVEWBVJVR
     VFVJUOVTTVSUPWBVEEVHVFVJUQSURUSWAUTVAVCVPTVD $.
 
+  $( Excluded middle is equivalent to the power set of ` 1o ` being finite .
```
Comment (benjub) [inline iset.mm:66241]:
```suggestion
  $( Excluded middle is equivalent to the power set of ` 1o ` being finite.
```

**Review by benjub — APPROVED** [review]
It is probably worth adding the theorem
```
( EXMID <-> A. x e. ~P 1o DECID x = 1o )
```
That is: if, in the set of truth values (the powerset of 1o), equality to 1o is decidable, then excluded middle holds (and conversely).

File: iset.mm (hunk 1 above)
Comment (jkingdon) [reply iset.mm:66241]: Changed; thanks.

**Comment by jkingdon** [comment]
> It is probably worth adding the theorem
> 
> ```
> ( EXMID <-> A. x e. ~P 1o DECID x = 1o )
> ```
> 
> That is: if, in the set of truth values (the powerset of 1o), equality to 1o is decidable, then excluded middle holds (and conversely).

I notice that you didn't say that it had to be me who adds this (which I appreciate).

But in this case, I'll take it on. Added to this pull request (although we can always split it out into a separate one if there is reason to do that).

**Review by avekens — APPROVED** [review]

**Review by benjub — APPROVED** [review]

**Review by tirix — APPROVED** [review]

**Pull request merged**